src/components/userPreferencesModal/UserPreferencesModal.styl

```diff
@@ -0,0 +1,152 @@
+@import './../../design/styles/common/navbar.styl'
+@import './../../design/styles/common/global.styl'
+
+.nav
+  padding-left: 0;
+  list-style: none;
+
+  > li
+    position: relative;
+    display: block;
+
+  > li
+    float: left;
+    margin-bottom: -1px;
+
+.modal-open
+  overflow: hidden;
+
+.modal
```

Comment:
Do we have a more generic "modal" component that should contain these?

Reply:
I didn't find it, we can create a more generic style file for modal css and use it here, what do you think?